assets/js/state/sagas/databases.test.js

```diff
@@ -32,4 +38,26 @@ describe('SAP Systems sagas', () => {
       })
     );
   });
+
+  it('should remove the database', async () => {
+    const { sap_system_id: id, sid } = databaseFactory.build();
+
+    const dispatched = await recordSaga(databaseDeregistered, {
+      payload: { id, sid },
+    });
+
+    expect(dispatched).toContainEqual(
+      removeDatabase({
+        id,
+        sid,
```

Comment:
It doesn't look like `sid` is necessary for `removeDatabase()`. A minor thing though...